src/timbot.py

```diff
@@ -175,13 +175,13 @@ def main():
 
     # connect to database
     try:
-        conn = mysql.connector.connect(user=os.environ['SQL_USER'],
-                                       password=os.environ['SQL_PASSWORD'],
-                                       host=os.environ['SQL_HOST'],
-                                       database=os.environ['SQL_DATABASE'])
-
-    except Exception as err:
-        print(err)
-        sys.exit()
+        conn = mysql.connector.connect(user=os.environ['MYSQL_USER'],
+                                       password=os.environ['MYSQL_PASSWORD'],
+                                       host=os.environ['MYSQL_HOST'],
+                                       database=os.environ['MYSQL_DATABASE'])
+        print("Established a connection to the MYSQL database instance")
+    except mysql.connector.Error as e:
```

Comment:
So the reason I changed this initially was b/c if you have any of the forth variables not set it will crash timbot

This either needs to
1. Be a generic catch
2. Have a separate catch block for variables not being defined
3. We initialize those variables as empty values **then** populate them with the OS environment values

Reply:
Yeah good point, not sure how I missed that locally when I was testing this. It's almost like we could mock this scenario pretty easily 8). Updated to just catch both exceptions for now and in the case of the connector error, we actually get something decent returned to us. In the case the type of error that gets raised is not of type mysql.connector.Error or KeyError, I think it's fine for timbot to just panic and exit early as that's something we should be able to catch pretty quickly.